bindings/nodejs/examples/how_tos/accounts_and_addresses/create_account.ts

```diff
@@ -47,10 +58,11 @@ async function run() {
             process.env.NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_1,
         );
 
+        // Create a new account
         const account = await wallet.createAccount({
-            alias: 'Alice',
+            alias: `${process.env.ACCOUNT_ALIAS_1}`,
```

Comment:
This should also work
```suggestion
            alias: process.env.ACCOUNT_ALIAS_1,
```

Reply:
Same in some other examples